packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -813,13 +806,6 @@ void main() {
     );
 
     expect(find.byType(EditableText), paints
-      ..rrect(
```

Comment:
Can you explain why these two  `..rrect`s had to be removed?

Reply:
In fact, this PR will fix two differences. 

First, the blue(Color 0xbf2196f3) cursor no longer blinks while dragging. Second, when dragging, the gray(Color 0xff999999) cursor that moves between text is also removed. These are the differences between the Flutter and the iOS native.
These two `..rrect`s validate the previous grey(Color 0xff999999) cursor at the bottom. 

![image](https://user-images.githubusercontent.com/7621572/172568408-4bc3e933-db69-47b9-b3c9-f7f83feb85f2.png)

Reply:
Got it, thanks for the clear explanation with screenshots.